app/controllers/admin/communities_controller.rb

```diff
@@ -9,11 +9,7 @@ def getting_started
     @selected_left_navi_link = "getting_started"
     @community = @current_community
 
-    if(feature_enabled?(:onboarding_redesign_v1))
-      redirect_to admin_getting_started_guide_path
-    else
-      render locals: {paypal_enabled: PaypalHelper.paypal_active?(@current_community.id)}
-    end
+    redirect_to admin_getting_started_guide_path
```

Comment:
We could now remove this whole action and also the related `getting_started.haml` (and all the related translations)